src/hooks/useAuthenticatedFetch.js

```diff
@@ -0,0 +1,38 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { useCallback } from "react";
+
+/**
+ * Get an 'authenticatedFetch' function which, if the user is logged in,
+ * will automatically attach an access token to any API request.
+ * @returns {{authenticatedFetch:(path:string, opts:Record)=>Promise}}
+ */
+export function useAuthenticatedFetch() {
```

Comment:
__accolade:__ I appreciate the use here of an authenticated fetch function more generally, that is then utilized by the authenticated API call function